src/main/java/com/inversoft/rest/RESTClient.java

```diff
@@ -351,6 +351,20 @@ public URI getURI() {
     return this;
   }
 
+  /**
+   * Add URL parameters from a {@code Map<String, Object>}.
+   *
+   * @param urlParameters The url parameters <code>Map</code> to add.  For each item in the <code>Map</code>
+   *                      this will call <code>urlParameter(String, Object)</code>
+   * @return This.
+   */
+  public RESTClient<RS, ERS> urlParameter(Map<String, Object> urlParameters) {
```

Comment:
My comment was actually to rename the method itself, but the parameter is also a good change. 

Should be :

```
public RESTClient<RS, ERS> urlParameters(Map<String, Object> urlParameters) {
   // Code
}
```

Reply:
Oops.  I misunderstood what var needed changing.  I just pushed an update for this.